client/components/docs-example/index.jsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Count from 'components/count';
+
+const DocsExample = ( {
```

Comment:
I like functions a lot, just my rule of thumb is that if I need comments to separate different function sections, probably the function is too long.